config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts do
```

Comment:
- [Optional] Dear @kambereBr, Great job setting up your routes and creating a working app. The issue I noticed from your route is related to how you ignore the `only: [:index, :show]` for the `post` route. This way of creating routes will create all routes that we are not currently using. We need to stay professional and only create the routes we need.

Reply:
Thanks, dear @TadesseDev for your review 👏 

`Routes` issue fixed.